### PR TITLE
Rewrite MyAccount using React Context and useReducer

### LINKED ISSUE #82: Yellow warning status bar comes everytime we change node settings, even if an account already selected
Change to only show if localStorage doesn't have any accounts saved.

### PR DESCRIPTION
**Pull request by siman**
Fix https://github.com/Joystream/apps/issues/82

### CODE REVIEW

**Comment by mnaamani** [comment]
Overall it looks good and seems to solve the original problem.
One thing doesn't seem to be working correctly though, if I have one key and I "forget" it, I would have expected the topbar to re-appear but instead I still see a key selected, and "joy.myAddress" is stil in localstorage.

**Comment by mnaamani** [comment]
After forgetting last key:

Screen Shot 2019-04-30 at 4 53 56 PM

**Comment by mnaamani** [comment]
ok I see I haven't pulled down last commit :)

File: packages/apps/src/Content/index.tsx [hunk 1]
```diff
@@ -69,7 +69,8 @@ class Content extends React.Component<Props> {
     }
 
     return (
-      <Wrapper><Provider>
+      <Wrapper>
+        <MyAccountProvider>
```
Comment (mnaamani) [inline packages/apps/src/Content/index.tsx]: Is it possible to have context Providers as children of a context provider?

If you are planning to make use of this redux style state management (which I like very much btw) and the answer to the above question is no, then the MyAccountProvider (being so high up in the component layout hierarchy) will need a better name.

Comment (siman) [reply packages/apps/src/Content/index.tsx]: > Is it possible to have context Providers as children of a context provider?

Yes, absolutely! Any context provider can have as many children context providers as needed.

Comment (mnaamani) [reply packages/apps/src/Content/index.tsx]: ok great, and presumably the child components can access the context they care about by setting the `contextType` and react will make the context available from the relevant provider?

**Pull request merged**